core_types/src/entry/entry_type.rs

```diff
@@ -85,6 +85,7 @@ impl EntryType {
     pub fn is_app(&self) -> bool {
         match self {
             EntryType::App(_) => true,
+            EntryType::Deletion => true,
```

Comment:
?
Why this? A deletion entry is a "system" entry type, not an app entry..

Reply:
Woops, This stayed form me debugging. Will change that asap